File: 240323/길이 단위 변환하기/change-length-unit.cpp
#include <iostream>

int main() {
    double one_fit=30.48;
    double one_mi=160934;

    std::cout<<std::fixed;
    std::cout.precision(1);

    std::cout<<"9.2ft = "<<9.2*one_fit<<"cm"<<std::endl;
    std::cout<<"1.3mi = "<<1.3*one_mi<<"cm";
}

File: 250210/길이 단위 변환하기/change-length-unit.java
public class Main {
    public static void main(String[] args) {
        //1fit=30.48cm,1mi=160934cm
        double oneFit=30.48;
        double oneMi=160934;

        double a = 9.2;
        double b = 1.3; 

        System.out.printf("%.1fft = %.1fcm\n",a,a*oneFit);
        System.out.printf("%.1fmi = %.1fcm",b,b*oneMi);
    
    }
}